site/templates/default.html

```diff
@@ -12,6 +12,38 @@
       <link rel="stylesheet" href="/css/hl.css">
       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/glider-js@1/glider.min.css">
       <script src="https://cdn.jsdelivr.net/npm/glider-js@1/glider.min.js"></script>
+      <script>
+         (function () {
+            try {
+              var preferredUwu;
+              try {
+                preferredUwu = localStorage.getItem('uwu');
+              } catch (err) { }
+
+              const isUwuValue = window.location 
+                && window.location.search
+                && window.location.search.match(/uwu=(true|false)/);
+                
+              if (isUwuValue) {
+                const isUwu = isUwuValue[1] === 'true';
+                if (isUwu) {
+                  try {
+                    localStorage.setItem('uwu', true);
+                  } catch (err) { }
+                  document.documentElement.classList.add('uwu');
+                  console.log('uwu mode enabled. turn off with ?uwu=false')
+                  console.log('logo credit to @sawaratsuki1004 via https://github.com/SAWARATSUKI/ServiceLogos');
```

Comment:
Not blocking, but it might be better to have a separate page for acknowledgements on the website.